src/org/rascalmpl/library/util/Monitor.rsc

```diff
@@ -11,28 +11,175 @@
 @bootstrapParser
 module util::Monitor
 
+import util::Math;
+import IO;
 
-@synopsis{Log the __start__ of a job.}
+@synopsis{Log the start of a job.}
 @description{
 jobStart registers a new current job on the job stack with an amount of
 steps todo and how much work it contributes (when it ends) to its parent job (if any).
 }
 @javaClass{org.rascalmpl.library.util.Monitor}
-public java void jobStart(str label, int work=1, int totalWork=100);
+java void jobStart(str label, int work=1, int totalWork=0);
```

Comment:
Maybe we should revert this, but `1` would be a more workable solution than `100`. I'll put `100` back because this is an accidental debugging change.